jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -288,4 +298,8 @@ private Variant loadVariantWhenAuthorized(
         // unauthorized...
         return null;
     }
+
+    private String extractPushIdentifier(HttpServletRequest request) {
+        return request.getHeader("push-identifier");
```

Comment:
@matzew Well, it's a header now, so let's keep it as it and agree on a name , I like your suggestion "aerogear-push-id" , @edewit you agree ?